package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cypress",
+  "name": "@you54f/cypress",
```

Comment:
We'll be unable to accept these changes to the package.json name and url. Do you have some autoupdate on your precommit or IDE that changes this?

Reply:
Hey, totally understand and  sorry didn’t mean to add those changes to this PR, for the url and package name. They were changed so I could test the changes with a packaged product  🙂

I don’t think you will want to accept the changes in the pr until new edge moves from beta release, so I think this pr might be a bit of a a slow burner

Reply:
Hey, I've now removed the erroneous `package.json` changes 👍 thanks for updating the title too.